packages/icons/svgr.config.js

```diff
@@ -20,6 +21,19 @@ export const ${componentName} = ({
   ${jsx}
 )
 
+const showOutline = ({ focusable, tabIndex }) => {
+  if (focusable === 'true' && parseInt(tabIndex, 10) >= 0){
+    return ''
+  }
+  else {
+    return 'outline: none;'
+  }
+}
+
+export const ${componentName} = styled(BaseComponent)\`
```

Comment:
2 Questions:

1. Why are non-alphanumeric characters being escaped here?
2. Should this code also be added to packages/icons/src/Icon.js?

Reply:
1) it broke on nested backticks, this was the only thing that worked - since we are inside the template backticks that start on line 9 https://github.com/pricelinelabs/design-system/blob/Fix-outline-property-for-icons/packages/icons/svgr.config.js#L9

2) Icon uses these svgr-built components, so it should be there - https://github.com/pricelinelabs/design-system/blob/Fix-outline-property-for-icons/packages/icons/src/Icon.js#L8

Reply:
it's ugly - if you have a better way of rendering backticks and `${}` inside of the template, without evaluation, please suggest.

Reply:
Ok, this makes sense.  I approve!